java-checks/src/main/java/org/sonar/java/checks/ArrayForVarArgCheck.java

```diff
@@ -66,7 +65,7 @@ public void visitNode(Tree tree) {
       MethodInvocationTree mit = (MethodInvocationTree) tree;
       sym = mit.symbol();
       args = mit.arguments();
-      methodName = mit.methodSelect();
+      methodName = ExpressionUtils.methodName(mit);
```

Comment:
This is a change of behavior : this should imply another refactoring or a test modification.

Reply:
Agreed. I was hesitating for this one. I'm going to discard it.